.github/ISSUE_TEMPLATE/issue report.yml

```
@@ -87,46 +144,42 @@ body:
     validations:
       required: true
   - type: markdown
+    id: general-information-logs
     attributes:
       value: |
-        When providing logs, please keep the following things in mind.
+        When providing logs, please keep the following things in mind:
         1. **DO NOT** use external paste services.
```

```suggestion
        1. **DO NOT** use external paste services. If logs are too large to paste into the field, upload them as text files in the Attachments field below
```

Should also add a field below for people to upload other attachments that aren't screenshots

---

Such as? They can add text files if the log is too large. There is also an "additional information" field that can be used.